l1-contracts/test/Rollup.t.sol

```diff
@@ -25,6 +28,7 @@ contract RollupTest is DecoderBase {
   Outbox internal outbox;
   Rollup internal rollup;
   MerkleTestUtil internal merkleTestUtil;
+  TxsDecoderHelper internal txsHelper;
```

Comment:
Added the helper here because I needed the `computeNumTxEffectsToPad` here to correctly pad l2 to l1 messages.